devolving_music/lib/Song_Scores.py

```diff
@@ -0,0 +1,39 @@
+from devolving_music.lib.elo_scoring import elo_rating
+from devolving_music.models.song_submission import SongSubmission
+from collections.abc import Iterable
+
+class SongScores():
+    def __init__(self,submission:SongSubmission):
```

Comment:
```suggestion
    def __init__(self, submission: SongSubmission):
```

Reply:
https://docs.python.org/3/library/collections.abc.html#collections.abc.Iterable

i thought it was fine but maybe I was wrong

Reply:
wait I think I was wrong so yeah good catch